2023-round-1/devotuoma/nx_tutorial_script.py

```diff
@@ -0,0 +1,28 @@
+# importing networkx library
+import networkx as nx
+# printing the version once more
+print(nx.__version__)
+G = nx.DiGraph() # calling on the diagraph function using G as the global variable
+
+#incorporating nodes with attributes group and color based on types
+G.add_node(1, group='int',color='red')
+G.add_node("Outreachy", group='int',color='green')
+G.add_node((2,4), group='tuple',color='blue')
+G.add_node(3, group='int',color='red')
+
+#adding edges
+G.add_edges_from([(1,3),("Outreachy",(2,4)),(3,"Outreachy"),((2,4),1)])
+
+#getting the shortest_path
+shortest_path = nx.shortest_path(G)
+
+#printing the source and destination of each possible paths 
+for s in shortest_path:
+    print("shortest paths from", s, ":")
+    for d in shortest_path[s]:
+        print( "to", d,":", shortest_path[s][d])
+
+colors = nx.get_node_attributes(G,'color').values() #getting the color attributes of each node
+
+nx.draw_networkx(G,with_labels=True,node_color=colors) #drawing the graph 
```

Comment:
Thanks @devotuoma for the pull request.
This looks very similar to another script https://github.com/networkx/outreachy/blob/main/2023-round-1/Qudirah/nx_tutorial_script.py 

Could you rewrite the script from scratch? You can use the networkx tutorial https://networkx.org/documentation/latest/tutorial.html to learn more about how to create graphs and run algorithms. Thanks!

Reply:
Am going to work on this.